src/chef-server-ctl/plugins/psql.rb

```diff
@@ -62,25 +61,36 @@
     db_config.merge!(JSON.parse(File.read("/etc/opscode-reporting/opscode-reporting-running.json")))
   end
 
-  seed=known_dbs[service_name]["hashseed"]
-  db_hash_key=known_dbs[service_name]["config_key"]
-  db_name=known_dbs[service_name]["dbname"]
+  seed = known_dbs[service_name]["hashseed"]
+  db_hash_key = known_dbs[service_name]["config_key"]
+  db_name = known_dbs[service_name]["dbname"]
 
   # undocumented, but available. Intended for use primarily for
   # gather-logs to be able to do its thing correctly.
   if (ARGV.include? "--as-admin")
     cfg = running_service_config('postgresql')
-    db_username=cfg['db_superuser']
-    db_password=credentials.get('postgresql', 'db_superuser_password')
+    db_username = cfg['db_connection_superuser'] || cfg['db_superuser']
+    db_password_string = 'db_superuser_password'
```

Comment:
As we talked about in the review, it would be worth it to try and simplify our use cases here; can we reduce it to one or two modes (e.g. admin, and per database user (oc_erchef)

Reply:
Can discuss findings in the next review.